pkg/kubelet/kubelet.go

```diff
@@ -2082,6 +2153,8 @@ func (kl *Kubelet) podIsTerminated(pod *api.Pod) bool {
 	return false
 }
 
+// filterOutTerminatedPods returns the given pods which the status manager
+// considers failed or succeeded.
```

Comment:
This function does the opposite ... it  returns a list of pods which are *not* considered failed or succeeded. 

Also, "status manager" sounds like a implementation detail, no? The function actually calls `podIsTerminated()` to find out the status.